com.bestaticpy.bestatic.yml

```diff
@@ -0,0 +1,12 @@
+id: com.bestaticpy.bestatic
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: bestatic
+modules:
+  - python3-Bestatic.json
```

Comment:
the main module should be in this file not in a separate one. The automatically generated mainifest for the python dependencies however should be kept in a separate file for easy regeneration.

Reply:
I do not have any other module than the python3-Bestatic. We just had to edit a bit that what flatpak-pip-generator produced as it could not resolve all the PyPI dependencies correctly. Do you want me to move anything specific from json file to yaml file?

Reply:
You should move the metainfo file and icon installation steps + sources here. Those will get overwritten when regenerating with pip generator.

Reply:
Done.